packages/SystemsThinkingWorkbench-Test.package/STWConnectionTest.class/instance/testSuccesfulPlacement.st

```diff
@@ -14,5 +14,4 @@ testSuccesfulPlacement
 	
 	connection connect: node1 with: node2.
 	
-	self assert: (connection vertices = {50@radius. 50@(50-radius)}).
-	
+	"TODO: Test bezier, but how?"
```

Comment:
That is not really possible, i would move these UI testing questions to the xcom for now.